### ISSUE TITLE
Pip install fails for perception_dataset

### ISSUE THREAD

**Issue by kminoda** [tier4/tier4_perception_dataset#148]
# Description

Pip install for perception_dataset fails
```bash
15.51 ERROR: Cannot install perception-dataset and perception-dataset==1.0.14 because these package versions have conflicting dependencies.
15.51 
15.51 The conflict is caused by:
15.51     perception-dataset 1.0.14 depends on nuscenes-devkit<2.0.0 and >=1.1.9
15.51     t4-devkit 0.1.0 depends on nuscenes-devkit<2.0.0 and >=1.1.11
15.51 
15.51 To fix this you could try to:
15.51 1. loosen the range of package versions you've specified
15.51 2. remove package versions to allow pip attempt to solve the dependency conflict
15.51 
15.51 ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

Even after applying this comming: https://github.com/tier4/tier4_perception_dataset/commit/be0af6fcf34fc20d67572b76cc3cb9ac49a95f03, I still get the following error

```bash
14.02 ERROR: Ignored the following versions that require a different python version: 0.36.0 Requires-Python >=3.6,<3.10; 0.37.0 Requires-Python >=3.7,<3.10; 0.52.0 Requires-Python >=3.6,<3.9; 0.52.0rc3 Requires-Python >=3.6,<3.9; 0.53.0 Requires-Python >=3.6,<3.10; 0.53.0rc1.post1 Requires-Python >=3.6,<3.10; 0.53.0rc2 Requires-Python >=3.6,<3.10; 0.53.0rc3 Requires-Python >=3.6,<3.10; 0.53.1 Requires-Python >=3.6,<3.10; 0.54.0 Requires-Python >=3.7,<3.10; 0.54.0rc2 Requires-Python >=3.7,<3.10; 0.54.0rc3 Requires-Python >=3.7,<3.10; 0.54.1 Requires-Python >=3.7,<3.10; 1.6.2 Requires-Python >=3.7,<3.10; 1.6.3 Requires-Python >=3.7,<3.10; 1.7.0 Requires-Python >=3.7,<3.10; 1.7.1 Requires-Python >=3.7,<3.10
14.02 ERROR: Could not find a version that satisfies the requirement rerun-sdk<0.17.0,>=0.16.1 (from t4-devkit) (from versions: none)
14.02 ERROR: No matching distribution found for rerun-sdk<0.17.0,>=0.16.1
14.02 
14.02 [notice] A new release of pip is available: 23.0.1 -> 24.2
14.02 [notice] To update, run: pip install --upgrade pip
------

```

# How to reproduce

Use this Dockerfile
```dockerfile
FROM python:3.10-slim-buster
ENV PYTHONUNBUFFERED 1
RUN apt-get update && apt-get install -y git
ARG perception_dataset_url=https://github.com/tier4/tier4_perception_dataset.git
ARG perception_dataset_version=fix/nuscenes_version
RUN git clone -b ${perception_dataset_version} ${perception_dataset_url} \
    && cd tier4_perception_dataset && pip install .
```

**Comment by kminoda** [comment]
Seems the issue is due to the base image. I've solved the problem by using ubuntu:22.04 image instead.

**Comment by kminoda** [comment]
Related: https://github.com/rerun-io/rerun/issues/1320
